aict_tools/apply.py

```diff
@@ -249,12 +249,12 @@ def apply_cuts_cta_dl1(
                 selection_config,
                 n_events=len(table),
             )
-            for row, match in zip(table.iterrows(), mask):
-                n_rows_before += 1
-                if match:
-                    remaining_showers.add((row['obs_id'], row['event_id']))
-                    new_table.append([row[:]])
-                    n_rows_after += 1
+            n_rows_before = len(table)
```

Comment:
Wouldn't `n_rows_before += len(table)` be better, to see it for all tels? Otherwise it just prints it for the last tel i think